internal.h

```diff
@@ -86,13 +98,18 @@ void fcgi_stop (void);
 /* Rest */
 extern bool rest_use_arrays;
 extern bool rest_use_types;
-gboolean rest_init (const char *path);
+gboolean rest_init (const char *path, const char *logging);
```

Comment:
I dont think this needs to be done in rest.c ... the logging init/shutdown can be done in main.c

Reply:
Done